browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -102,6 +102,15 @@ function _createRadioButton(parentContainer, treeViewData, builder, entry) {
 	return radioButton;
 }
 
+function _createColorPreviewButton(parentContainer, builder, entry) {
+	var colorPreviewButton = L.DomUtil.create('input', builder.options.cssClass + ' ui-treeview-checkbox', parentContainer);
+	colorPreviewButton.type = 'color-preview';
```

Comment:
I don't think it is correct value. I think we can only use standard types.
https://www.w3schools.com/html/html_form_input_types.asp

Probably it should be an img